sre-operational-fabric.md

```diff
@@ -0,0 +1,71 @@
+# (Remote) SRE - Rackspace Private Cloud
+
+As a Rackspace Private Cloud (RPC) Operational Fabric SRE, you will help
+support our organizational vision: To become the preferred brand for consuming
+complex open source infrastructure, as-a-Service, delivering industry-leading
+simplicity, efficiency, and reliability.
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in datacenters throughout
+the world.
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focused on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+## About Us
+
+- **Team Size**: Currently 8 senior developers/engineers and a technical manager
+- **Hours**: We have flexible working hours and aim for a median of 35-40 hours a week
+- **On-Call**: No, we have a 24x7 ops team and escalations (uncommon) go to management
+- **Location**: US/UK work from home with offices in US/UK (most states supported)
+- **Travel**: About once or twice a year for a conference or team gathering
+
+## Responsibilities
+
+- Help develop and maintain *observability*/controllability of customer environments
+- Help drive zero downtime (0DD) CI/CD for our teams' infrastructure and app stack
+- Work directly with support engineers, operations, product management, and end-users
+- Develop automation to improve efficiency, reliability, and simplicity of clouds
+- Take high level problems and autonomously deliver solutions supported by data
+- Develop solutions that leverage and integrate existing open source technologies
+- Engage asynchronously with team members via Slack, Jira, github, and email
+- Engage synchronously with team members via video calls
+
+## Qualifications
+
+The ideal candidate will have the following (**not** hard requirements):
+
+- Experience defining architecture of production systems that run at scale
+- Experience developing and deploying applications at scale
+- Experience working on distributed teams
+- Experience with 0DD CI/CD on Kubernetes using Helm
```

Comment:
Personally, I wouldn't mention Helm. We already have Jed who AIUI is a Helm expert. Maybe something generic like "orchestration tooling"

Reply:
Ack, gonna leave it